### PR TITLE
Support relationship type as str in `gds.graph.relationshipProperties.stream`

### PR DESCRIPTION
**Pull request by orazve**
The problem from issue #554 was reproduced locally and showed that `gds.graph.relationshipProperties.stream` has different API in Cypher and Python Client sides: Cypher [can get](https://neo4j.com/docs/graph-data-science/current/management-ops/graph-reads/graph-stream-relationships) `list[str]` as `relationshipType`, but Python Client [can get](https://neo4j.com/docs/graph-data-science-client/1.10/api/graph/#gds.graph.relationshipProperties.stream) `str` and `list[str]`.

This PR contain:

* Fix for issue [#554](https://github.com/neo4j/graph-data-science-client/issues/554) - usage `gds.graph.relationshipProperty.stream` without calling `separate_property_columns=True` parameter
* Update PyG test version to get all metrics for test step
* Add tests for `gds.graph.relationshipProperties.stream` with relationshipType as `str`

**Question**: why this problem is not reproduced by CI, where we run `kge-predict-transe-pyg-train.ipynb` which calls `gds.graph.relationshipProperties.stream` function with relationshipType as `str`, not with `list[str]`?

Before submitting this PR, please read [Contributing to the Neo4j Ecosystem](https://github.com/neo4j/graph-data-science-client/blob/main/CONTRIBUTING.md). 

Make sure:
- [x] You signed the [Neo4j CLA](https://neo4j.com/developer/cla/#sign-cla) (Contributor License Agreement) so that we are allowed to ship your code in our library
- [x] Your contribution is covered by tests

### CODE REVIEW

File: graphdatascience/tests/integration/test_graph_ops.py [hunk 1]
```diff
@@ -652,7 +652,7 @@ def test_graph_streamRelationshipProperties_with_arrow(gds: GraphDataScience) ->
 
 
 @pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 2, 0))
-def test_graph_relationshipProperties_stream_with_arrow(gds: GraphDataScience) -> None:
+def test_graph_relationshipProperties_stream(gds: GraphDataScience) -> None:
```
Comment (brs96) [inline graphdatascience/tests/integration/test_graph_ops.py:655]: I think we want to keep 'with_arrow` in the name. It uses `gds`, that has arrow enabled. `gds_without_arrow` is the one without.

File: graphdatascience/tests/integration/test_graph_ops.py [hunk 2]
```diff
@@ -671,7 +671,7 @@ def test_graph_relationshipProperties_stream_with_arrow(gds: GraphDataScience) -
     assert {e for e in y_values["propertyValue"]} == {5, 6, 7}
 
 
-def test_graph_streamRelationshipProperties_with_arrow_separate_property_columns(gds: GraphDataScience) -> None:
+def test_graph_streamRelationshipProperties_separate_property_columns_dep(gds: GraphDataScience) -> None:
```
Comment (brs96) [inline graphdatascience/tests/integration/test_graph_ops.py:674]: I think we want to keep `with_arrow` in the name. It uses `gds`, that has arrow enabled. `gds_without_arrow` is the one without.

File: graphdatascience/tests/integration/test_graph_ops.py [hunk 3]
```diff
@@ -685,7 +685,7 @@ def test_graph_streamRelationshipProperties_with_arrow_separate_property_columns
 
 
 @pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 2, 0))
-def test_graph_relationshipProperties_stream_with_arrow_separate_property_columns(gds: GraphDataScience) -> None:
+def test_graph_relationshipProperties_stream_separate_property_columns(gds: GraphDataScience) -> None:
```
Comment (brs96) [inline graphdatascience/tests/integration/test_graph_ops.py:688]: same here with test name

File: graphdatascience/graph/graph_entity_ops_runner.py [hunk 4]
```diff
@@ -70,13 +70,6 @@ def _handle_properties(
         )
 
 
-class GraphElementPropertyRunner(GraphEntityOpsBaseRunner):
```
[the remark this answers is not in the document]
Comment (orazve) [reply graphdatascience/graph/graph_entity_ops_runner.py:73]: Yes, so it was renamed, moved closer to `GraphRelationshipPropertiesRunner` and added wrapping into a list if string was passed.

**Review by brs96 — APPROVED** [review]

File: graphdatascience/graph/graph_entity_ops_runner.py [hunk 5]
```diff
@@ -197,6 +190,16 @@ def drop(self, G: Graph, node_properties: List[str], **config: Any) -> "Series[A
         ).squeeze()
 
 
+class GraphRelationshipPropertyRunner(GraphEntityOpsBaseRunner):
+    @compatible_with("stream", min_inclusive=ServerVersion(2, 2, 0))
+    def stream(
+        self, G: Graph, relationship_property: str, relationship_types: Strings = ["*"], **config: Any
+    ) -> DataFrame:
+        self._namespace += ".stream"
+        relationship_types = [relationship_types] if isinstance(relationship_types, str) else relationship_types
```
[the remark this answers is not in the document]
Comment (orazve) [reply graphdatascience/graph/graph_entity_ops_runner.py:199]: Thanks, I've made an experiment where I put this wrapping into a list in `_handle_properties` and it breaks `nodeProperty.stream` and `relationshipProperty.stream` cases. And I think we won't need this wrapping when we align APIs after getting this card done.
So I prefer to leave it as it is now and merge.

**Pull request merged**